build_torch_xla_libs.sh

```diff
@@ -35,7 +35,7 @@ if [[ "$XLA_BAZEL_VERBOSE" == "1" ]]; then
 fi
 
 BUILD_STRATEGY="standalone"
-if [[ "$XLA_SANDBOX_BUILD" == "0" ]]; then
```

Comment:
Maybe we can just do `if [[ "$XLA_SANDBOX_BUILD" == "1" ]]` and then a `else`？

Reply:
Lol sounds good, done.